install/mysql/glpi-empty.sql

```diff
@@ -7375,6 +7381,7 @@ CREATE TABLE `glpi_tickettemplates` (
   `entities_id` int unsigned NOT NULL DEFAULT '0',
   `is_recursive` tinyint NOT NULL DEFAULT '0',
   `comment` text,
+  `status_limit` varchar(255) NOT NULL DEFAULT '[1,2,3,4,5,6]',
```

Comment:
Maybe I am wrong, but I found `status_limit` ambiguous. Indeed, people may think that the template only applies when item has a status that matches given values.

Maybe field could be named `allowed_statuses`, to ensure there is no ambiguity.